Formula/g/git-cola.rb

```diff
@@ -35,7 +35,15 @@ class GitCola < Formula
   end
 
   def install
+    python = Formula["python@3.11"].opt_bin / "python3"
+    ENV["PYTHON3"] = python
+    ENV["PYTHON"] = python
+
```

Comment:
If `virtualenv_install_with_resources` already handles setting `$PATH` so that the first `python3` in the `$PATH` is python3.11 then these two environment variables are not strictly needed. I added them because I have random python interpreters in my environment and I wanted to make sure that the `make git-cola.app` step uses homebrew's.

Let me know if this is redundant and I'll remove the lines above.